lib/main.dart

```diff
@@ -1,125 +1,141 @@
 import 'package:flutter/material.dart';
 
+import 'custom_widgets/middle_text.dart';
+import 'custom_widgets/title_widget.dart';
+import 'entities/Movie.dart';
+import 'ui_constants.dart';
+import 'text_styles/custom_text_style.dart';
+
 void main() {
   runApp(const MyApp());
 }
 
 class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
-  // This widget is the root of your application.
+  final Movie movie = const Movie(
+      'The Super Mario Bros. Movie',
+      'The Super Mario Bros. Movie',
+      'https://image.tmdb.org/t/p/w500/qNBAXBIQlnOThrVvA6mA2B5ggV6.jpg',
+      'https://image.tmdb.org/t/p/w500/9n2tJBplPbgR2ca05hS5CKXwP2c.jpg',
+      '2023-04-05',
+      '7,8',
+      'While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi '
+          'are transported down a mysterious pipe and wander into a magical new world. But when the '
+          'brothers are separated, Mario embarks on an epic quest to find Luigi.');
```

Comment:
You can move this implementation to Movie class as a factory method, to avoid creating this kind of data on UI side

Reply:
done